static/install/web.html

```diff
@@ -35,6 +35,16 @@
     <body>
         {% include "header.html" %}
         <main id="web-install">
+            <p id="sticky-error-text" hidden="hidden">Your device doesn't have enough free storage space.
+                This issue is likely due to "Incognito mode", "InPrivate Window" or similar
+                private browsing feature. which restricts how much storage is available to the web
+                installer. Please make sure you have enough free storage and you are not using private browsing feature,
+                check <a href="#prerequisites">prerequisites</a> section.
+            </p>
```

Comment:
Suggestion: something like: "The Shields function of the Brave browser restricts the amount of file storage that a web page can use.  This will cause the installer to fail.  If you have not disabled Shields for this web site, please close this window, disable Shields for this site, and start over.  Also, please check that all necessary #prerequisites are satisfied."